lib/vehicle_factory.rb

```diff
@@ -0,0 +1,16 @@
+class VehicleFactory
+
+  def initialize; end
+
+  def create_vehicles(database)
+    database.map do |vehicle|
```

Comment:
Nice job using `.map`.

Reply:
thanks, I took a lot from our enumerable's lesson, i'm also biased because i'm not a fan of .each do.